syntax = "proto2";

message Book {
    required string ISBN = 1;
    required string title = 2;
    required string author = 3;
    required string genre = 4;
    required int32 publishingYear = 5;
}